.changes/nextrelease/simplify-foreach-with-in-array

```diff
@@ -0,0 +1,7 @@
+[
+    {
+        "type": "enhancement",
+        "category": "Aws",
```

Comment:
The Aws namespace is added by default, so the category here should just be "Waiter".